packages/chonky/src/components/file-list/GridEntryPreview.tsx

```diff
@@ -41,17 +41,11 @@ export const GridEntryPreviewFolder: React.FC<FileEntryPreviewProps> = React.mem
         [externalClassName || '']: !!externalClassName,
     });
     return (
+        <><div id="gridFolderSelectionIndicator" className={commonClasses.selectionIndicator}></div>
         <div className={className}>
-            <div className={folderClasses.folderBackSideMid}>
-                <div className={folderClasses.folderBackSideTop} />
-                <div className={folderClasses.folderFrontSide}>
-                    <GridEntryDndIndicator className={fileClasses.dndIndicator} dndState={dndState} />
```

Comment:
I think we should keep the DndIndicator for now.. seems like it's an incomplete change. `GridEntryPreviewFile` still has it, and the functionality inside Chinky is still there, even if we don't have the indicator.

Reply:
I added the DnDIndicator back to folder preview card.